test/e2e/main_test.go

```diff
@@ -24,16 +24,18 @@ import (
 )
 
 var (
-	f                  = framework.Global
-	t                  *testing.T
-	namespace          string
-	nodes              []string
-	startTime          time.Time
-	bond1              string
-	bridge1            string
-	primaryNic         = flag.String("primaryNic", "eth0", "Primary network interface name")
-	firstSecondaryNic  = flag.String("firstSecondaryNic", "eth1", "First secondary network interface name")
-	secondSecondaryNic = flag.String("secondSecondaryNic", "eth2", "Second secondary network interface name")
+	f                    = framework.Global
+	t                    *testing.T
+	namespace            string
+	nodes                []string
+	startTime            time.Time
+	bond1                string
+	bridge1              string
+	primaryNic           = flag.String("primaryNic", "eth0", "Primary network interface name")
+	firstSecondaryNic    = flag.String("firstSecondaryNic", "eth1", "First secondary network interface name")
+	secondSecondaryNic   = flag.String("secondSecondaryNic", "eth2", "Second secondary network interface name")
+	nodesInterfacesState = make(map[string][]byte)
+	interfacesToIgnore   = []string{"flannel.1"}
```

Comment:
I think this is the same as veth we want to ignore them directly at knmstate [1]

[1] https://github.com/nmstate/kubernetes-nmstate/blob/master/deploy/operator.yaml#L78.

Reply:
I would not say so. There is only a couple of bridges on the host and not seeing a specific one is weird. On the other hand, with veths we are consistent by ignoring all of them.

Reply:
ack